warp10/src/main/java/io/warp10/script/functions/MAP.java

```diff
@@ -138,6 +141,14 @@ public Object apply(WarpScriptStack stack) throws WarpScriptException {
   }
 
   private Object applyWithParamsFromMap(WarpScriptStack stack, Map params) throws WarpScriptException {
+    if (params.containsKey(DEPRECATED_PARAM_OCCURENCES)) {
+      throw new WarpScriptException(getName() + " was given a deprecated parameter: " + DEPRECATED_PARAM_OCCURENCES + ". Use instead parameter " + PARAM_OCCURRENCES);
+    }
+
+    if (params.containsKey(DEPRECATED_PARAM_TICKS)) {
+      throw new WarpScriptException(getName() + " was given a deprecated parameter: " + DEPRECATED_PARAM_TICKS + ". Use instead parameter " + PARAM_OCCURRENCES);
+    }
```

Comment:
Can you add something like "parameter name and behavior are changed, read the doc"
It is not a simple "replace".